translations/monero-core_it.ts

```diff
@@ -140,37 +140,37 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="117"/>
         <source>Close</source>
-        <translation type="unfinished">Chiudi</translation>
+        <translation>Chiudi</translation>
     </message>
     <message>
         <location filename="../components/DaemonConsole.qml" line="128"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished">comando + enter (per esempio help)</translation>
+        <translation>comando + enter (esempio: help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="93"/>
         <source>Starting Monero daemon in %1 seconds</source>
-        <translation type="unfinished">Partenza Monero daemon in %1 secondi</translation>
+        <translation>Il daemon partirà tra %1 secondi</translation>
     </message>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="115"/>
         <source>Start daemon (%1)</source>
-        <translation type="unfinished">Start daemon (%1)</translation>
+        <translation>Start daemon (%1)</translation>
     </message>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="132"/>
         <source>Use custom settings</source>
-        <translation type="unfinished">utilizza settaggi custom</translation>
+        <translation>utilizza impostazioni personalizzate</translation>
     </message>
 </context>
 <context>
     <name>DaemonProgress</name>
     <message>
         <source>Synchronizing blocks %1/%2</source>
-        <translation type="vanished">Sincronizzazione blocchi %1/%2</translation>
```

Comment:
I was suggesting the whole translated message be removed because it is no longer needed.
Same with the other `type="vanished"` below.